### PR TITLE
update fedora icon

### PR DESCRIPTION
**Pull request by stefanobartoletti**
Updated Fedora Linux Logo and brand color, as in their website https://getfedora.org/

The source of the SVG file is from [this link](https://pagure.io/fedora-logos/blob/master/f/fedora/fedora_logo.svg), marked as the source for the [logo on Wikipedia](https://commons.wikimedia.org/wiki/File:Fedora_logo_(2021).svg)

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for the contribution, @stefanobartoletti. Did you intend to close this PR, though?

We're actually waiting on official approval from Fedora to update our icon so, if you'd still be interested in taking it on then please leave a comment in #5540 and we will assign the issue to you and let you know when we've heard back from them.

**Comment by stefanobartoletti** [comment]
Yes, I closed this intentionally because I noticed the discussion in the issue you mentioned only after, and I was under the impression that their stance was already clear.

If you want you can repoen it, if they give their approval I'll gladly take care of this.

**Comment by sachinraja** [comment]
Hi @stefanobartoletti, we now have legal approval from Fedora to add this icon! I'll reopen this and make a few suggestions on the data, but the icon matches perfectly already.

**Review by mattdm — CHANGES_REQUESTED** [review]
Please also add 

            "guidelines": "https://fedoraproject.org/wiki/Legal:Trademark_guidelines",
            "license": {
                "type": "custom",
                "url": "https://docs.fedoraproject.org/en-US/project/brand/"

File: _data/simple-icons.json
Comment (sachinraja) [inline _data/simple-icons.json:2844]:
```suggestion
            "source": "https://docs.fedoraproject.org/en-US/project/brand/"
```

**Review by sachinraja — CHANGES_REQUESTED** [review]

**Review by sachinraja — APPROVED** [review]
made the changes to merge this so we can get this in, thanks @stefanobartoletti

**Pull request merged**
